util/file_util.py

```diff
@@ -540,7 +548,7 @@ def file_load_csv_dict(
     config: dictionary of aggregation settings in case there are multiple rows
       with the same key. refer to aggregation_util.aggregate_dict for config
       settings.
-
+    key_index: if True, each rows is loaded with a unique key for row index.
```

Comment:
Per implementation, `key_index` gets preference over `key_column`. Should we disallow key_column when key_index = True?

Reply:
Added an error log if key_column and key_index are both set.

Reply:
Sorry, I am not able to locate error log. IMO, error logs are easy to miss. Throwing precondition error (ValueError) might be a tighter enforcement.